packages/social-previews/src/twitter-preview/style.scss

```diff
@@ -1,112 +1,225 @@
-	width: 125px;
-	height: 125px;
-}
-
-.twitter-preview__large_image_summary .twitter-preview__image {
-	width: 506px;
-	max-width: 100%;
-	height: 254px;
-	border-bottom: 1px solid #e1e8ed;
-}
-
-.twitter-preview__body {
-	padding: 0.75em;
-	text-decoration: none;
-	font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
-	font-size: $font-body-small;
-	color: black;
-	text-align: left;
-	line-height: 1.3em;
-	overflow: hidden;
-}
-
-.twitter-preview__summary .twitter-preview__body {
-	margin-left: 125px;
-	border-left: 1px solid #e1e8ed;
-	height: 100%;
-}
-
-.twitter-preview__large_image_summary .twitter-preview__body {
-	padding-left: 1em;
-	padding-right: 1em;
-}
-
-.twitter-preview__title {
-	max-height: 1.3em;
-	white-space: nowrap;
-	font-weight: bold;
-	font-size: 1em;
-	margin: 0 0 0.15em;
-	overflow: hidden;
-	text-overflow: ellipsis;
-}
-
-.twitter-preview__description {
-	margin-top: 0.32333em;
-	max-height: 3.9em;
-	text-overflow: ellipsis;
-	//clamp after two lines
-	display: -webkit-box;
-	-webkit-line-clamp: 2;
-	-webkit-box-orient: vertical;
-	overflow: hidden;
-}
-
-.twitter-preview__summary .twitter-preview__description {
-	-webkit-line-clamp: 3;
-}
-
-.twitter-preview__url {
-	text-transform: lowercase;
-	color: #8899a6;
-	max-height: 1.3em;
-	white-space: nowrap;
-	overflow: hidden;
-	text-overflow: ellipsis;
-	margin-top: 0.32333em;
+.twitter-preview__container {
+	display: grid;
+	grid-template-columns: 65px auto;
+	margin-bottom: 5px;
+	margin-right: 24px;
+
+	.twitter-preview__sidebar {
+		display: grid;
+		grid-template-rows: 35px auto;
+		justify-items: center;
+
+		.twitter-preview__profile-image {
+			img {
+				height: 30px;
+				width: 30px;
+				border-radius: 15px;
+				object-fit: cover;
+			}
+		}
+
+		.twitter-preview__connector {
+			width: 2px;
+			background-color: #8c8f94;
+		}
+	}
+
+	.twitter-preview__name {
+		font-weight: bold;
+		font-size: 16px;
+		line-height: 19px;
+	}
+
+	.twitter-preview__screen-name,
+	.twitter-preview__date {
+		color: #667886;
+		font-size: 16px;
+		line-height: 18px;
+		letter-spacing: -0.3px;
+		margin-left: 15px;
+	}
+
+	.twitter-preview__content {
+		margin: 7px 0;
+
+		.twitter-preview__text {
+			font-size: 14px;
+			line-height: 18px;
+			letter-spacing: -0.3px;
+			color: #787c82;
+			white-space: pre-wrap;
+		}
+
+		.twitter-preview__media {
+			border-radius: 15px;
+			overflow: hidden;
+			display: grid;
+			grid-gap: 2px;
+			grid-template-areas: 'a';
+			height: 300px;
+			margin-top: 10px;
+
+			img,
+			video {
+				width: 100%;
+				height: 100%;
+				object-fit: cover;
+
+				&:nth-child( 1 ) {
+					grid-area: a;
+				}
+				&:nth-child( 2 ) {
+					grid-area: b;
+				}
+				&:nth-child( 3 ) {
+					grid-area: c;
+				}
+				&:nth-child( 4 ) {
+					grid-area: d;
+				}
+			}
+
+			&.twitter-preview__media-children-2 {
+				grid-template-areas: 'a b';
+			}
+
+			&.twitter-preview__media-children-3 {
+				grid-template-areas: 'a b'
+									 'a c';
```

Comment:
Is this formatting right? It looks really strange here on Github

Reply:
The alignment indenting got converted to tabs at some point. 🙂 Fixed in c2af41cd0a.